scripts/images/README.md

```diff
@@ -47,7 +50,7 @@ where:
 -d:  Image distribution; coreos, rhel, centos
 -U:  Redhat Subscription username(for RHEL)
 -P:  Redhat Subscription password(for RHEL)
--O:  OS Password (for RHEL and CentOS)
+-O:  [OBSOLETE, but required for now, specify anything] OS Password (for RHEL and CentOS)
```

Comment:
It will be set for the OS password for the image

Reply:
Will drop this change in my next push. 👍